flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/copy/HiveParserBaseSemanticAnalyzer.java

```diff
@@ -2106,7 +2111,7 @@ public static CatalogBaseTable getCatalogBaseTable(
     public static class TableSpec {
         public ObjectIdentifier tableIdentifier;
         public String tableName;
-        public CatalogBaseTable table;
+        public ResolvedCatalogBaseTable<?> table;
```

Comment:
Do we really need `ResolvedCatalogBaseTable`?  What I mean is can we avoid to call the method `catalogManager.resolveCatalogBaseTable`? It's a internal method, we always want to avoid to call it.

Reply:
I think we need the table to be resolved since we want `validatePartColumnType` on this, it requires the resolved type information.